app/lib/fetch-token.ts

```diff
@@ -2,8 +2,26 @@ type TokenResponse = {
   token: string;
 };
 
-export const fetchToken = async (apiKeyAuthentication = true) => {
-  return apiKeyAuthentication
-    ? 'xx697404a7-6cfd-48c6-93d1-30d73d17e07a' // demo api key
-    : ((await (await fetch('http://localhost:3000/token')).json()) as TokenResponse).token;
+export const fetchToken = async (request?: null | Request, apiKeyAuthentication = false) => {
+  if (apiKeyAuthentication) {
+    return 'xx697404a7-6cfd-48c6-93d1-30d73d17e07a'; // demo API key
```

Comment:
I know it was like that before/you did not add it yourself, but that's not a very good thing to document as an example to follow (hardcoded the API key directly in the code) 😁